docs/CommandGuide/handshake-runner.md

```diff
@@ -0,0 +1,71 @@
+% handshake-runner(1) | "handshake-runner" Documentation
```

Comment:
Nice.  What is this first line?

Reply:
It is a metadata block for pandoc. I use an [internet resource](https://eddieantonio.ca/blog/2015/12/18/authoring-manpages-in-markdown-with-pandoc/) to write a markdown to manpage convertible document using pandoc. I think handshake-runner doesn't need to have a manpage so should drop this line and the next `NAME`  line.